rlclientlib/multistep.cc

```diff
@@ -5,7 +5,7 @@
 namespace reinforcement_learning
 {
 void episode_history::update(
-    const char* event_id, const char* previous_event_id, string_view context, const ranking_response& resp)
```

Comment:
Have you removed it since it is not used?

Reply:
yes. Found it unused.

Reply:
The reason why it was there is the following:
Although right now history is just single trivial class it supposed to be interface + multiple implementations. And interface (in order to be generic enough) should depend on response (since it is also part of history).
I am fine with removing it for now, but just wanted to emphasize that it will probably get back once we make progress in that direction.

Reply:
If you want to let me add it back in this PR or later, that is fine for me. I was thinking it should be the episode history being sent to decision stacks, not the ranking_response being sent to the history to be altered.

 For this topic discussion, I sent one email earlier "[Episodic Learning] Episode history design". If you could comment on that, it will be helpful.

Reply:
Sorry for delay, can we actually keep ranking_response in update signature in order not to forget this dependency?